app/views/watches/show.html.erb

```diff
@@ -1,12 +1,15 @@
-<!-- navbar -->
-<div class="navbar-lewagon">
-
-</div>
-<!-- watch banner -->
-
-<div class="banner">
-  <img src="" alt="">
-</div>
-<!-- name -->
-<!-- price -->
-<!-- book now form -->
+
+<div class= "navbar-lewagon"></div>
```

Comment:
we are using a navbar in the show page 😉? or are you just interested in the `.navbar-lewagon` class?